content/latest/deploy/kubernetes/helm-chart.md

```diff
@@ -192,6 +192,32 @@ REVISION  UPDATED                   STATUS    CHART           DESCRIPTION
 1         Fri Oct  5 09:04:46 2018  DEPLOYED  yugabyte-latest Install complete
 ```
 
+## Connect using YCQL client
+
+By default YugaByte helm will expose only the master ui endpoint via LoadBalancer. If you wish to expose YCQL, YEDIS and YSQL services via LoadBalancer for your app to use, you could do that via the following command.
+
+```sh
+helm install yugabyte -f expose-all.yaml --namespace yb-demo --name yb-demo --wait
+```
+
+One can exec into one of the yb-tserver containers and run the packaged cqlsh to get access to the shell. For example:
```

Comment:
Lets drop this way and just show how to connect using cqlsh on the load balancer directly. So move this section below the next where we get the external IP.

Also, change the text here to:

You can connect to this cluster with `cqlsh` as follows: